infer_segmentation.py

```diff
@@ -1,6 +1,11 @@
 import os
 import shutil
+import random
+random.seed(1)
 import numpy as np
+np.random.seed(12)
+from tensorflow import set_random_seed
+set_random_seed(123)
```

Comment:
these I wouldn't set up here in import - this is super easy for someone to overlook and be real confused where they are set. would suggest making these command line arguments w/ these default values